README.md

```diff
@@ -52,6 +52,12 @@ Note:  The URL of the product to store the results for, in the format of
 Note: this was tested against the docker container
   docker pull codechecker/codechecker-web:6.15.2
 
+To select different analyzer you can use:
+
+    CODECHECKER_ANALYZER = "clangsa clang-tidy"
```

Comment:
Is the clang static analyzer already included in the dependencies or does it need something extra?

Reply:
@bachp  should be but I will do a test on clean environment before to confirm

Reply:
@bachp Thank you to let me test it properly. I was very concentrate on qt5 applicaiton and c++ code based on cmake and make but forget to test simple hello world. I done it now and fixed in the first patch. The clangsa come for free on meta-clang. I was able to run both